pom.xml

```diff
@@ -60,8 +59,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>10</source>
```

Comment:
This is 

        <plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-compiler-plugin</artifactId>
          <version>${maven-compiler.version}</version>
          <configuration>
            <encoding>UTF-8</encoding>
            <showDeprecation>true</showDeprecation>
            <release>${java.version}</release>
            <debug>true</debug>
          </configuration>
        </plugin>

with 
<java.version>11</java.version>
<maven-compiler.version>3.12.1</maven-compiler.version>

see https://github.com/SonarOpenCommunity/sonar-cxx/blob/master/pom.xml